### DISCUSSION TITLE
自定义主页：PCL 新功能说明书

### DISCUSSION THREAD

**Discussion by WForst-Breeze** [Meloong-Git/PCL#4905] [自定义主页]
内容：本主页会在 PCL **近一个大版本** 的更新内容中，挑出足以较大影响用户使用体验的 **主要功能更新**，并给予用户对应功能的 **使用方法**。使不愿意看长篇幅更新日志的用户可以用最少的时间 **了解并学会** 使用 PCL 更新的新功能。同时页面下方添加了 PCL 社区的动态，便于使更多用户参与到贡献中来（毕竟现在多语言翻译人手实在紧缺…… 未来这里还会放一些其它的需要社区玩家一同贡献完成的内容的，也算是活跃活跃社区，多个人多份力嘛~ 但是如果觉得不合适我可以删掉）

作者：风释清然SC
仓库：https://github.com/WForst-Breeze/Whats-New-PCL2
联网更新地址：`https://gitee.com/wforstbreeze/whats-new-pcl/raw/main/Custom.xaml`
预览：
![QQ截图20241005130914](https://github.com/user-attachments/assets/2fb805c1-69d2-4fa3-87d1-b7ba6273b1c2)
<details>
<summary>展开后样式</summary>

![QQ截图20241005133258](https://github.com/user-attachments/assets/fa1db295-b0c0-45b2-a8ca-7a47b44305b9)

</details>

#### 其他想说的话
本主页以简洁干净为主，没有标注作者、版权等信息，排版和措辞原则遵循帮助库规范，符合原生 PCL 风格，已经测试在小窗口中可以正常显示。（换言之，有些更新的功能没有必要放进帮助库，但是可以放在这个主页作为引导 orz）
等高三结束回来再把联网更新地址从 Gitee 换掉吧 orz，现在实在没有太多精力，抱歉！

**Comment by LTCatt** [comment]
可以考虑把关于卡片合并到下方的更多里面去，它在最上方有点占用空间了。
更新内容可以降序排列，把最新的排在最上面。
已经在 2.8.9 加进去了！

**Reply by WForst-Breeze** [reply]
十分感谢 www！

提到的问题只能在下周末解决了 orz 抱歉…… 如果周中有时间会抽时间改一改的 🤓

**Comment by youzi-3222** [comment]
无意间看到，`查看完整更新日志` 按钮会被右下角工具按钮挡掉，感觉可以调整一下（

![image](https://github.com/user-attachments/assets/c1e0e4b5-2ecf-4124-999c-2cd68fece5c2)
